test/01_basic/cibuildwheel_test.py

```diff
@@ -0,0 +1,49 @@
+import subprocess, sys, os
+from glob import glob
+import utils
+
+def test():
+    project_dir = os.path.dirname(__file__)
+
+    # build the wheels
+    subprocess.check_call([sys.executable, '-m', 'cibuildwheel', project_dir]) 
```

Comment:
Minor idea: what about moving this into `utils` as well? `utils.call_cibuildwheel()` or so? This gets rid of `import subprocess, sys, os` as well, and then the test has even more focus on the things we're checking.

Reply:
(OK, I now see you'd need an `env` parameter, or `add_env`, with a dictionary of things to update)

Reply:
yes! I was just now working on something very similar....

![image](https://user-images.githubusercontent.com/1244307/56602942-7f15b500-65f6-11e9-8f00-96eb148f10e0.png)

Reply:
Yesss, great :-)

For me this takes away the burden that `cibuildwheel` isn't called from `run_test.py`, if it's just a single line to write anywa.